### ISSUE TITLE
Check if contrast ratio of all contents are > 4.5 (AA)

### ISSUE THREAD

**Issue by RaenonX** [RaenonX-DL/dragalia-site-front#257]
According to Google Chrome Dev Tool, contrast ratio should be 4.5+ (AA); 7+ (AAA) is better.

Could use `LightHouse` for check?

**Comment by RaenonX** [comment]
Mostly done during #249.
